frontend/components/bc/MiniParser.vue

```diff
@@ -0,0 +1,226 @@
+<script setup lang="ts">
```

Comment:
I think I understand your opposition if this is something that you never saw in the companies you worked for: it looks non-standard.
Does it make it bad?

Maybe we could sit and talk about it rationally? I am not against good practice, I just want to understand what disadvantage is stronger than the benefits.

>  that much complexity

Your example demonstrates that a parser is less complex. You suggest:
```json
"title": "My title",
"list": {
  "0": ["first list item",  "a bold text",  "another normal text"],
  "1": ["second list item",  "a bold text",  "another normal text"],
}
```
but the json with a parser is simply:
```json
[
  "My title",
  "- first list item *a bold text* another normal text",
  "- second list item *a bold text* another normal text"
]
```

> will be harder to maintain / translate / adapt in the future

The current situation consists in hard-coded spans structures in the templates and list structures in the translation files, and some of them might be invalid when we translate to other languages. (+ we need hacks to silence eslint warnings)

You suggest to create a component for lists. It would come in addition to `formatMultiSpan` for bold and italic. Then we will we create a component for links...
Basically those things are "hard-coded parsers" and each of them requires complicated structures in the translation files and take the risk to be invalid with some languages and inject HTML into the VDOM.

I would like to understand rationally what is the problem with a parser that brings simplicity and cleanliness to the code and the json.